docs/reference/tags/form_tags/clear_cart/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Clear Cart
+parent: Tags
+has_children: false
+---
+
+The Clear Cart Tag renders a button that clears the user's cart. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart", return_to: config.clear_cart_link.url %}
+	 <input type="submit" value="clear" class="cart__clear {{id}}__clear" />
+ {% endform %}
+```
+{% endraw %}
+
+##### output
+{% raw %}
+```html
+<form action="/sites/cart" accept-charset="UTF-8" method="post">
+    <input type="hidden" name="_method" value="delete" autocomplete="off">
+    <input type="hidden" name="return_to" id="return_to" value="https://www.creator-website.com/book-tickets" autocomplete="off">
+    <input type="submit" value="Clear" class="button btn--outline cart-block__cart-clear">
+</form>
+```
+{% endraw %}
+
+##### redirect
+On button click, the Clear Cart Tag will redirect to the cart page by default.
+To specify the redirect location, pass a URL as the value of the return_to: param.
+
+The following example redirects to the homepage. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart", return_to: 'http://www.creator-site.com/home' %}
+	 <input type="submit" value="clear" class="cart__clear {{id}}__clear" />
```

Comment:
Given those classes aren't rendered by the tag, and aren't important for this to work, I'd remove it.